mpmp/src/main/Main.java

```diff
@@ -13,10 +13,11 @@
  */
 public class Main {
 	public static void main(String[] args) {
-		if(args.length < 2)
+		if(args.length < 1)
 			usage();
 
-		switch(args[1]) {
+		// 0th argument is not the command in Java -oki
```

Comment:
Yes, but I didn't really understood you, it was a tad too loud in the room and my mind wasn't focused.